test/test_other.py

```diff
@@ -8194,6 +8194,25 @@ def test_binaryen_ignore_implicit_traps(self):
     # sizes must be different, as the flag has an impact
     self.assertEqual(len(set(sizes)), 2)
 
+  @uses_canonical_tmp
+  @parameterized({
+    # In a simple -O0 build we do not set --low-memory-unused (as the stack is
+    # first, which is nice for debugging but bad for code size (larger globals)
+    # and bad for the low-memory-unused trick.
+    '': ([], False),
+    # When we optimize, we do.
+    'O2': (['-O2'], True),
+    # But a low global base prevents it.
+    'O2_GB_512': (['-O2', '-sGLOBAL_BASE=512'], False),
+    # A large-enough global base allows it.
+    'O2_GB_512': (['-O2', '-sGLOBAL_BASE=1024'], True),
+  })
+  def test_binaryen_low_memory_unused(self, args, low_memory_unused):
+    with env_modify({'EMCC_DEBUG': '1'}):
```

Comment:
Adding `-v` to the command line is a cleaner why to do this I think (and generates less output)